### ISSUE TITLE
Stop using links for buttons and vice versa

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#138]
The "Create Plan" link looks like a button, but is actually a link. This can cause confusion for screen reader users, as screen reading software allows the user to bring up a list of links on a page, as well as a list of just buttons. This "Create Plan" item can actually use the <button> tag, since it's performing an action. This will solve any confusion issues.

From old pivotal ticket: https://www.pivotaltracker.com/story/show/131647237

**Comment by briri** [comment]
Just focus on non org-admin/super-admin pages for this first pass

**Comment by briri** [comment]
I've looked through some explanations about this and found that its appropriate to use a link if the user will be directed away from the current page. If the clicking the element does something but the user is not redirected then a button should be used.

So, a form should have a submit button if the user is returned to the existing page and a link for 'cancel' if they are returned elsewhere.

**Comment by xsrust** [comment]
What exactly is the desired outcome from this ticket?
Looking at the documentation on [aria buttons](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role), would it be acceptable to just add the 'button' role to any links which do not redirect the user to another page?  This is the approach I am currently taking with this.

I also wanted to know if replacing submit buttons with accessible_submit_button's is within the scope of this ticket, or if it will be the subject of another ticket.
@briri

**Comment by briri** [comment]
I think that we can go with the role attribute for now. They will provide recommendations to us after the test if there are any issues with it. 

We should ideally use the accessible submit button but they each require a bit of JS. I was going to open a separate ticket for that after the accessibility test.

**Comment by xsrust** [comment]
ok, thanks.  I'll continue on with the role attribute (and aria-labelledby/aria-label) for the buttons then.

**Comment by briri** [comment]
@xsrust I added the `role="button"` attribute to all of the popover links on the forms while I was in adding aria tags for the font-awesome icons.

**Comment by stephaniesimms** [comment]
Closing this pending next round of accessibility testing
